src/langsmith/read-local-experiment-results.mdx

```diff
@@ -0,0 +1,174 @@
+- `blocking=True` (default): Blocks until all evaluations complete before returning. When you iterate over the results, all data is already available.
+
+Both modes return the same `ExperimentResults` type; the difference is whether the function waits for completion before returning. Use `blocking=False` for streaming and real-time debugging, or `blocking=True` for batch processing when you need the complete dataset.
+
+The following example demonstrates `blocking=False`. It iterates over results as they stream in, collects them in a list, then processes them in a separate loop:
+
+```python
+from langsmith import Client
+import random
+
+client = Client()
+
+def target(inputs):
+    """Your application or LLM chain"""
+    return {"output": "MY OUTPUT"}
+
+def evaluator(run, example):
+    """Your evaluator function"""
+    return {"key": "randomness", "score": random.randint(0, 1)}
+
+# Run evaluation with blocking=False to get an iterator
+streamed_results = client.evaluate(
+    target,
+    data="MY_DATASET_NAME",
+    evaluators=[evaluator],
+    blocking=False
+)
+
+# Collect results as they stream in
+aggregated_results = []
+for result in streamed_results:
+    aggregated_results.append(result)
+
+# Separate loop to avoid logging at the same time as logs from evaluate()
+for result in aggregated_results:
+    print("Input:", result["run"].inputs)
+    print("Output:", result["run"].outputs)
+    print("Evaluation Results:", result["evaluation_results"]["results"])
+    print("--------------------------------")
+```
+
+This produces output like:
+
+```
+Input: {'input': 'MY INPUT'}
+Output: {'output': 'MY OUTPUT'}
+Evaluation Results: [EvaluationResult(key='randomness', score=1, value=None, comment=None, correction=None, evaluator_info={}, feedback_config=None, source_run_id=UUID('7ebb4900-91c0-40b0-bb10-f2f6a451fd3c'), target_run_id=None, extra=None)]
+--------------------------------
+```
+
+## Understand the result structure
+
+Each result in the iterator contains:
+
+- `result["run"]`: The execution of your target function.
+  - `result["run"].inputs`: The inputs from your [dataset](/langsmith/evaluation-concepts#datasets) example.
+  - `result["run"].outputs`: The outputs produced by your target function.
+  - `result["run"].id`: The unique ID for this run.
+
+- `result["evaluation_results"]["results"]`: A list of `EvaluationResult` objects, one per evaluator.
+  - `key`: The metric name (from your evaluator's return value).
+  - `score`: The numeric score (typically 0-1 or boolean).
+  - `comment`: Optional explanatory text.
+  - `source_run_id`: The ID of the evaluator run.
+
+- `result["example"]`: The dataset example that was evaluated.
+  - `result["example"].inputs`: The input values.
+  - `result["example"].outputs`: The reference outputs (if any).
+
+## Example: Implement a quality gate
```

Comment:
nit/suggestion: move the example titles to H3, i.e., `## Example` and `### Implement a quality gate`